### PR TITLE
fix(1500): [2] add unzipArtifacts method

### PR DESCRIPTION
**Pull request by s-yoshika**
## Context
To implement SD_ZIP_ARTIFACTS feature by [new design](https://github.com/screwdriver-cd/screwdriver/blob/0962d20d251c0b7c03d40b7e4b10bb1ed009831d/design/sd-zip-artifacts.md)

Please refer the following for the progress of this feature.
https://github.com/screwdriver-cd/screwdriver/issues/1500#issuecomment-998382248

## Objective
This PR add a new method to request for queue-service to unzip artifacts.

note: this PR build should be failed until [executor-base](https://github.com/screwdriver-cd/executor-base/pull/75) will be updated.

## References
This PR depends on following PR.
- https://github.com/screwdriver-cd/executor-base/pull/75

[design doc](https://github.com/screwdriver-cd/screwdriver/blob/0962d20d251c0b7c03d40b7e4b10bb1ed009831d/design/sd-zip-artifacts.md#sd-api-api)

[issue](https://github.com/screwdriver-cd/screwdriver/issues/1500)

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -133,6 +133,24 @@ class ExecutorQueue extends Executor {
         return this.api(config, options);
     }
 
+    /**
+     * Unzip the ZIP of artifacts
+     * @method _unzipArtifacts
+     * @param  {Object}  config           Configuration
+     * @param  {Integer} config.buildId   Unique ID for a build
```
Comment (sakka2) [inline index.js]: It should be `Number` for js

File: index.js [hunk 2]
```diff
@@ -133,6 +133,24 @@ class ExecutorQueue extends Executor {
...
+     * @return {Promise}
```
Comment (sakka2) [inline index.js]: Don't we need to add `token` to param?
https://github.com/screwdriver-cd/models/blob/9b9d5bb9ca83d9f95cef2b44c9912198d31b65c5/lib/build.js#L682

Comment (s-yoshika) [reply index.js]: thanks, I missed.

**Review by sakka2 — APPROVED** [review]
LGTM

**Review by tk3fftk — APPROVED** [review]

**Review by ryoshimm — APPROVED** [review]
LGTM

**Review by tkyi — APPROVED** [review]

**Pull request merged**